conftest.py

```diff
@@ -0,0 +1,65 @@
+import gc
+import logging
+import os
+import time
+from pathlib import Path
+
+import psutil
+import pytest
+from _pytest.python import Function
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
lowercase?

Reply:
Well, constant should be in uppercase in python normally.
https://peps.python.org/pep-0008/#constants

Reply:
Yeah, but that’s a mutable instance of `logging.Logger`. `LOGGER` here is not a constant, but a global instance.

Reply:
E.g. similar case: https://github.com/pytorch/pytorch/blob/44f1c6e41c3cf636ba8f691fafbe3b4be621e718/torch/utils/_cuda_trace.py#L6C37-L6C37

Reply:
By the way, where do we actually use this?

Reply:
Removed, not used.